components/StandardApp.jsx

```diff
@@ -94,6 +94,20 @@ class AppInitComponent extends React.Component {
                     if (ConfigUtils.getConfigProp("dontLoadDefaultTheme")) {
                         return;
                     }
+
+                    // Determine default theme, and fallback if not available
+                    let availableThemes = themes.items.map((t) => t.id);
```

Comment:
Theme items might potentially be nested.

Reply:
Ok will try to come up with a commit that handles nested themes. Would you have a sample themes.json test with, or some info in how to create projects that have nested themes ? Is it with QGIS themes in the project ?

(In my themes.json, I see sublayers, but nothing looking like nested themes, not even an empty list)